llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -23,5 +23,7 @@ let TargetPrefix = "spv" in {
   def int_spv_store : Intrinsic<[], [llvm_i32_ty, llvm_anyptr_ty, llvm_i16_ty, llvm_i8_ty], [ImmArg<ArgIndex<2>>, ImmArg<ArgIndex<3>>]>;
   def int_spv_extractv : Intrinsic<[llvm_any_ty], [llvm_i32_ty, llvm_vararg_ty]>;
   def int_spv_insertv : Intrinsic<[llvm_i32_ty], [llvm_i32_ty, llvm_any_ty, llvm_vararg_ty]>;
+  def int_spv_extractelt : Intrinsic<[llvm_any_ty], [llvm_any_ty, llvm_anyint_ty]>;
```

Comment:
Can't we just live with G_[EXTRACT/INSERT]_VECTOR_ELT instead of the new intrinsics?